_episodes/introduction.md

```diff
@@ -175,33 +175,34 @@ the website we will be creating.
 
 Click the green "New" repo button on the left hand side of github
 
-![Green New Repo Button](../fig/new_repo_button.png)
+<img src="../fig/new_repo_button.png" alt="Green New Repo Button" width="100">
 
 **or** click the "+" menu in the upper righthand corner and
 choose "New Repository".
 
-![Dropdown plus menu with new repository option highlighted](../fig/plus_new_repo.png)
+<img src="../fig/plus_new_repo.png" alt="Dropdown plus menu with new repository option highlighted" width="200">
 
 Next you will need to fill in some info about your repository.
 
-![Blank new repository page](../fig/blank_new_repo.png)
+<img src="../fig/blank_new_repo.png" alt="Blank new repository page" width=600>
```

Comment:
This one looks a bit smaller than the others, although you used consistent width...

Reply:
I'm guessing there is slightly more whitespace around this screenshot